crates/bindings-csharp/Runtime/LogStopwatch.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SpacetimeDB
```

Comment:
Please remove unused usings (I believe none of these should be necessary? VSCode warns me about them, but maybe VS doesn't?) and use file-scoped namespace for consistency with other code.

Reply:
Ah looks like it's because I didn't add `.editorconfig` to the solution, and VS can't find it otherwise.

These legacy solution workflows are annoying sometimes 😅 I'll add it in a PR.

Reply:
Done